docs/rackhd/api-usage-examples.rst

```diff
@@ -21,21 +21,26 @@ This example provides instructions on how to flash a BMC image on a Quanta (node
 2. Post the obm settings if they don't already exist for the node. An example on how to do this is shown in Section 7.1.8.1 here http://rackhd.readthedocs.io/en/latest/tutorials/vagrant.html#adding-a-sku-definition Section 7.1.8.1
 
 
-3. The firmware files need to be built into a SKU package
+3. Quanta-T41, Quanta-D51 Firmware upgrades:
```

Comment:
maybe the information of which platforms support firmware upgrade could be locate at the beginning of this section